h2o-py/h2o/model/metrics_base.py

```diff
@@ -1470,8 +1470,8 @@ def _plot_pr(self, server=False, save_to_file=None, plot=True):
             if plt is None:
                 return decorate_plot_result(figure=RAISE_ON_FIGURE_ACCESS)
             fig = plt.figure()
-            plt.xlabel('Recall (TP/(TP+FP))')
-            plt.ylabel('Precision (TPR)')
+            plt.xlabel('Recall (TP/(TP+TN))')
```

Comment:
Shouldn't this say Recall (TP/TP + _F_N)? @wendycwong

Reply:
Looks like the same issue is in R:
https://github.com/h2oai/h2o-3/blob/8002ed8ff61916f5bb6527250e4d37aed39da46f/h2o-r/h2o-package/R/models.R#L4382

Reply:
<img width="289" alt="image" src="https://user-images.githubusercontent.com/7231712/162545899-96960c42-4f4c-4424-83dd-bdd303897be1.png">